assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -69,23 +69,17 @@ type ContribAttrs = {
 const subHeadingMonthlyText = {
   GB: 'from £5 a month',
   US: 'from $5 a month',
-};
-
-const subHeadingMonthlyTextTwo = {
-  GB: 'from £2 a month',
-  US: 'from $3 a month',
+  AU: 'from $5 a month',
 };
 
 function getSubHeadingMonthly(abTests: Participations, isoCountry: IsoCountry) {
-  return abTests.usRecurringAmountsTestTwo === 'range'
-    ? subHeadingMonthlyTextTwo[isoCountry]
-    : subHeadingMonthlyText[isoCountry];
+  return subHeadingMonthlyText[isoCountry];
 }
```

Comment:
👍 for getting rid of the test code.

Since all this function does now is an object lookup, could we make use of some referential transparency (🙂) and remove it?